app/src/main/java/com/marverenic/music/utils/SlsMessenger.java

```diff
@@ -0,0 +1,87 @@
+package com.marverenic.music.utils;
+
+import android.content.Context;
+import android.content.Intent;
+
+import com.marverenic.music.R;
+import com.marverenic.music.model.Song;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * A helper class for interacting with Simple Last.fm Scrobbler. The API description could be found
+ * <a href=https://github.com/tgwizard/sls/blob/master/Developer's%20API.md>here</a>.
```

Comment:
The rest of the project generally has the convention of inserting the URL directly instead of using an HTML tags, since HTML Javadocs aren't actually generated for this project.